static/src/javascripts/projects/commercial/modules/dfp/Advert.js

```diff
@@ -22,7 +22,7 @@ const createSizeMapping = (attr: string): Array<AdSize> =>
             size => (size === 'fluid' ? 'fluid' : size.split(',').map(Number))
         );
 
-const getAdBreakpointSizes = (advertNode: Element): AdSizes =>
+const getAdBreakpointSizes = (advertNode: HTMLElement): AdSizes =>
```

Comment:
@katebee I had to convert this type to `HTMLElement` as `advertNode` is eventually passed to Toggles constructor (in render-advert.js),  if nothing's passed to Toggles constructor it fallbacks to use `document.body` as the default, which is an `HTMLElement` type, therefore flow started complaining about an `Element` type being passed to it.

Changing `advertNode` from `Element` to `HTMLElement` hasn't caused any flow warnings.